packages/paste-core/components/avatar/src/utils.ts

```diff
@@ -45,7 +45,46 @@ export const getCorrespondingFontSizeFromSizeToken = (size: IconSizeOptions): Fo
   }
 };
 
-export const getComputedTokenNames = (size: IconSize): {lineHeight: LineHeight; fontSize: FontSize} => {
+export const getCorrespondingIconSizeFromSizeToken = (size: IconSizeOptions): IconSizeOptions => {
+  if (typeof size === 'string' && size.includes('sizeIcon')) {
+    switch (size) {
+      case 'sizeIcon10':
+      case 'sizeIcon20':
+      case 'sizeIcon30':
+      case 'sizeIcon40':
+      default:
+        return 'sizeIcon10';
+      case 'sizeIcon50':
+      case 'sizeIcon60':
+        return 'sizeIcon20';
+      case 'sizeIcon70':
+        return 'sizeIcon30';
+      case 'sizeIcon80':
+        return 'sizeIcon40';
+      case 'sizeIcon90':
+        return 'sizeIcon50';
+      case 'sizeIcon100':
+        return 'sizeIcon70';
+      case 'sizeIcon110':
+        return 'sizeIcon80';
+    }
+  }
+  throw new Error('[Avatar]: size must be of type IconSizeOptions.');
+};
+
+// this function takes in a size and exports an object w/ lineheight and fontsize
+export const getComputedTokenNames = (
+  size: IconSize
```

Comment:
@SiTaggart we rebased our branch after you made the change to the IconSize type, but when we hover over `IconSize` vscode still shows "size10" and "sizeIcon10" options. Same with `size` and `IconSizeOptions` types. Is that expected behavior?
![Screen Shot 2021-03-22 at 1 23 22 PM](https://user-images.githubusercontent.com/75342690/112031801-1c92ff80-8b12-11eb-89a4-00f124719863.png)

Reply:
This line (72) in the testing file still satisfies typescript instead of causing a typescript error 
![Screen Shot 2021-03-22 at 1 43 37 PM](https://user-images.githubusercontent.com/75342690/112034224-a04deb80-8b14-11eb-9404-a2479bb5f20e.png)

Reply:
> @SiTaggart we rebased our branch after you made the change to the IconSize type, but when we hover over `IconSize` vscode still shows "size10" and "sizeIcon10" options. Same with `size` and `IconSizeOptions` types. Is that expected behavior?
> 
> ![Screen Shot 2021-03-22 at 1 23 22 PM](https://user-images.githubusercontent.com/75342690/112031801-1c92ff80-8b12-11eb-89a4-00f124719863.png)
> 
> 

It should work, but I'd need to test locally to be sure. One thing worth checking though, after a rebase and rebuild is VS Code can get itself into a funk. Just make sure the entire library built and restart the typescript server. Otherwise I can pull it down and see what's going on

Reply:
rebuilding and restarting the ts server fixed this 👍  thanks